packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -154,8 +154,11 @@ export default class FocusZone extends React.Component<FocusZoneProps> implement
     // Assign initial tab indexes so that we can set initial focus as appropriate.
     this.updateTabIndexes();
 
-    if (this.props.shouldFocusOnMount) {
-      this.focus();
+    if (this.props.defaultTabbableElement && typeof this.props.defaultTabbableElement === 'string') {
+      this._activeElement = this.getDocument().querySelector(this.props.defaultTabbableElement) as HTMLElement;
```

Comment:
For both changes, is there a reason why defaultTabbableElement isn't called as a function here too?

Reply:
it might be beacuse the code paths for string and function are actually different, but I would also like to understand it more

Reply:
That's a valid concern here! The reason why this is not being called here is because it is called inside the `updateTabIndexes` function that appears just before this in line 155.